gnat2goto/driver/tree_walk.ads

```diff
@@ -76,4 +76,5 @@ package Tree_Walk is
      return Symbol
    with Pre => Nkind (N) = N_Compilation_Unit;
 
+   type Irep_Array is array (Positive range <>) of Irep;
```

Comment:
Ahh, I see now in the next commit why we need that. Perhaps this commit should be squashed with the next one? Otherwise this doesn't make a lot of sense.

Reply:
Squashed.